master/docs/manual/cfg-www.rst

```diff
@@ -543,20 +543,32 @@ Here is an nginx configuration that is known to work (nginx 1.6.2):
             # you could use / if you use domain based proxy instead of path based proxy
             location /buildbot/ {
                 proxy_pass http://127.0.0.1:5000/;
+                proxy_set_header Host $host;
```

Comment:
I dont think all this complexity is even needed.
Here is what we are using in the nine.buildbot.net

https://github.com/buildbot/buildbot-infra/blob/master/roles/nginx/templates/proxy

I think the whole think could be changed as

    proxy_set_header HOST $host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto  $scheme;
    proxy_set_header X-Forwarded-Server  $host;
    proxy_set_header X-Forwarded-Host  $host;
    map $http_upgrade $connection_upgrade {
        default upgrade;
        ''      close;
    }
    location /buildbot/ {
        proxy_pass http://127.0.0.1:5000/;
        proxy_buffering off;
        proxy_read_timeout 600;

        # These three lines are be required for web socket proxying
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection $connection_upgrade;
    }